README.md

```diff
@@ -70,6 +70,7 @@ Inspired by [awesome-python](https://awesome-python.com).
 * [RDKit](http://www.rdkit.org/) - Open-Source Cheminformatics Software.
 * [QUIP](http://libatoms.github.io/QUIP/) - A collection of software tools to carry out molecular dynamics simulations.
 * [tsase](http://theory.cm.utexas.edu/tsase/) - The library which depends on ASE to tackle transition state calculations.
+* [OPEM](https://github.com/ECSIM/opem) - Open source PEM fuel cell simulation tool.
```

Comment:
Could you expand the PEM abbreviation so it's more accessible for non-experts?

Reply:
Done ;-)